Microsoft.Toolkit.Uwp.UI.Animations/ConnectedAnimations/Connected.cs

```diff
@@ -0,0 +1,426 @@
+// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media.Animation;
+
+namespace Microsoft.Toolkit.Uwp.UI.Animations
+{
+    /// <summary>
+    /// A helper class that allows Connected Animations to be enabled through XAML
+    /// </summary>
+    /// <seealso cref="ConnectedAnimationService"/>
+    public static class Connected
+    {
+        private static List<ConnectedAnimationProperties> _connectedAnimationsProps = new List<ConnectedAnimationProperties>();
+        private static Dictionary<string, ConnectedAnimationProperties> _previousPageConnectedAnimationProps = new Dictionary<string, ConnectedAnimationProperties>();
+        private static Dictionary<UIElement, List<UIElement>> _coordinatedAnimationElements = new Dictionary<UIElement, List<UIElement>>();
+
+        private static Frame _navigationFrame;
+
+        /// <summary>
+        /// Gets or sets the <see cref="Frame"/> hosting the pages where the Connected Animation will run
+        /// Set this value if the frame hosting your pages is different than the root frame of the Window
+        /// </summary>
+        public static Frame NavigationFrame
```

Comment:
I'm thinking an attached property on the Frame would be better, rather than a big static

Reply:
Great idea

Reply:
Done